// Copyright (c) 2020 The Orbit Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package orbit_grpc_protos;

message ModuleInfo {
  reserved 1;
  string file_path = 2;
  uint64 file_size = 3;
  uint64 address_start = 4;
  uint64 address_end = 5;
  string build_id = 6;
  uint64 load_bias = 7;
}
